.github/workflows/scripts/reclaim_disk_space.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# remove 4GiB of images
+sudo systemd-run docker system prune --force --all --volumes
+
+# remove unused software
+sudo systemd-run eatmydata -- rm -rf \
```

Comment:
Is this necessary? rm doesn't sync. I also don't see the point in doing systemd-run here instead of just detaching. AFAICT this whole file is just

```sh
#!/bin/sh
sudo docker system prune --force --all --volumes &
rm -rf "$A_T" /opt ... /v/l/snapd &
```

Reply:
I don't think that GitHub Actions give us guarantees that background processes won't be killed after finishing run script of a stage.

Reply:
Out of curiosity, how long does it take to run these commands in the foreground?  How much time does this save us?

Reply:
```
+ time sudo docker system prune --force --all --volumes
Deleted Images:
untagged: node:14
untagged: node@sha2
...
deleted: sha256:d05b8af4c7ce1e4ccbecd4bb29fa4ac82a5fdeee36ef41bc46549678058513e0
Total reclaimed space: 3.503GB
0.05user 0.07system 0:58.49elapsed
+ time sudo rm -rf /opt/hostedtoolcache /opt/az /opt/cni /opt/containerd /opt/google /opt/hhvm /opt/hostedtoolcache /opt/microsoft /opt/mssql-tools /opt/pipx /opt/pipx_bin /opt/post-generation /opt/runner /opt/vsts /usr/local/aws-cli /usr/local/aws-sam-cli /usr/local/bin /usr/local/doc /usr/local/etc /usr/local/games /usr/local/graalvm /usr/local/include /usr/local/julia1.7.1 /usr/local/lib /usr/local/man /usr/local/n /usr/local/sbin /usr/local/sessionmanagerplugin /usr/local/share /usr/local/sqlpackage /usr/local/src /usr/share/az_3.1.0.zip /usr/share/az_4.4.0.zip /usr/share/az_5.9.0.zip /usr/share/az_6.6.0.zip /usr/share/az_7.1.0 /usr/share/dotnet /usr/share/gradle-7.3.3 /usr/share/miniconda /usr/share/swift /var/lib/gems /var/lib/mysql /var/lib/snapd
1.69user 17.04system 4:25.35elapsed
```

~330 seconds altogether

Reply:
That's interesting.  The `rm` here is much slower than I would have suspected.  Let's go ahead and use `systemd-run` as proposed, these processes shouldn't be killed, but just in case.